### ISSUE TITLE
Setting arguments(..., ..., atleast1=false) still expects at least one argument

### ISSUE THREAD

**Issue by genotrance** [fenekku/commandeer#19]
```nim
import commandline

commandline:
    arguments(TEST, string, false)
```

Still expects at least one argument. Error is "Missing command line arguments".

**Comment by fenekku** [comment]
Thanks for raising this! I fixed it and pushed the fix under version 0.12.1. Let me know if it works on your side.

cheers,

**Comment by genotrance** [comment]
The fix worked. Thank you for the quick turnaround!
